doc/Performing_IO_Operations_Using_S3Client.rst

```diff
@@ -30,6 +30,12 @@ You can install the AWS CLI on the CORTX OVA to perform IO operations.
 
 #. Ensure that all the prerequisites are satisfied. See `Prerequisites <#Prerequisites>`__.
 
+#. Install the unzip package:
```

Comment:
@TechWriter-Mayur Can we move `yum install unzip -y` in Prerequisite section. So that you will not need to duplicate it. 
Also I do not see any `unzip` command in document. So just curious where it's used.

Reply:
Updated.